src/models/annotation/usePostAnnotation.js

```diff
@@ -14,8 +14,12 @@ export default function usePostAnnotation() {
     rect,
     viewpoint,
     theta = 0,
+    encounterGuid = null,
   ) => {
     try {
+      const additionalProperties = encounterGuid
```

Comment:
we should be migrating request hooks to `useMutate` instead of continuing to develop/modify with the existing hooks. if you can use an old one as-is that's okay i guess, but at the point where we are changing/enhancing them we really should be migrating

Reply:
Done.